docs/resources/ui-building-blocks/pages/page-elements.md

```diff
@@ -0,0 +1,414 @@
+  can also be customized with different widgets for a more tailored visual representation.
+- **Actions:** Hosts icon buttons for various operations like search, share, and more, situated on
+  the right end.
+
+### Add an AppBar to your Page
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/Gviwe4k9svWyMBr6NLCP?embed&show_copy_link=true"
+        title="Add AppBar"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+## Floating Action Button (FAB)
+
+**Floating Action Button (FAB)** is a distinctive circular button that hovers over content, commonly
+used for a primary action within an app, like adding a new item or composing a message.
+
+### Extended Property
+
+This variant of the `FAB` includes both an icon and a label, making it larger than the standard
+circular `FAB`. It is useful when you want the action button to convey more information than just the
+icon can provide, such as text explaining the action ("Add Task", "Create Post", etc.).
+
+**Use-cases**
+
+The **extended** `FAB` is particularly beneficial in applications where the action needs clear and
+immediate recognition from the user, which cannot be fully achieved by an icon alone. It is also
+useful in interfaces where there is ample space to accommodate a longer button without cluttering
+the UI.
+
+![fab-comparison.png](..%2Fimgs%2Ffab-comparison.png)
+
+### Adding a Floating Action Button to your Page
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/TfHpfAQYIc5iaALgbK2O?embed&show_copy_link=true"
+        title="Add FAB"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+## Drawers
+
+**Drawer** is a slide-out menu that can emerge from either side of the screen, typically used for
+app navigation or placing additional options. It allows users to switch between different sections
+of an app without cluttering the main interface.
+
+### Add a Drawer to your Page
```

Comment:
In the Arcade, maybe we can have a third slide that shows the hotspot on the drawer tile. Just like the third slide in the FAB arcade.

Reply:
Fixed!